scripts/gcp_setup/environment/gcloud_env.py

```diff
@@ -647,39 +689,20 @@ def generate_forseti_conf(self):
             os.path.join(
                 ROOT_DIR_PATH, 'configs', 'forseti_conf_dm.yaml'))
 
-        # Ask for SendGrid API Key
-        print('Forseti can send email notifications through SendGrid '
-              'via an API key. '
-              'This step is optional and can be configured later.')
-        sendgrid_api_key = raw_input(
-            'What is your SendGrid API key? (press [enter] to skip) ').strip()
-        if sendgrid_api_key:
-            self.sendgrid_api_key = sendgrid_api_key
+        self._get_user_input()
```

Comment:
I would like to see the ```if self.unattended_install:``` condition to be outside here, instead of being inside ```_get_user_input()```.  This way, it's easier to understand the workflow that if this is not an unattended_install, then prompt the user for input.

```
if not self.unattended_install:
    _get_user_input()
```